common/utils/bgettext/bgettext.c

```diff
@@ -21,6 +21,8 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 #include <string.h>
 
+enum { BGETTEXT_PLURAL = 0x01, BGETTEXT_CONTEXT = 0x02, BGETTEXT_DOMAIN = 0x04 };
```

Comment:
Just a minor nitpick - given that these are bit flags, you can be more explicit and use left shift operator to define them (as we usually do in such cases):
```
BGETTEXT_PLURAL = 1 << 0,
BGETTEXT_CONTEXT = 1 << 1,
```